isso/tests/test_comments.py

```diff
@@ -604,6 +604,115 @@ def testAddComment(self):
         self.app.db.comments.activate(1)
         self.assertEqual(self.client.get('/?uri=test').status_code, 200)
 
+    def testModerateComment(self):
+
+        id_ = 1
```

Comment:
Using `id_` instead of my usage of private `_id` is much smarter, good catch!

Reply:
its also what PEP8 tells you to do ;)